edx-solutions/xblock-ooyala
# [MCKIN-6858] Changed non-China friendly CDNs to China-friendly CDNs.

This Pull Request ensures that the ooyala player can be used in China by changing out Google CDN urls for China-Friendly CDN urls.

**JIRA tickets**: MCKIN-6858

**Merge deadline**: "None" 

**Testing instructions**:
1. ...
2. Open up Developer tools in the browser and confirm that the Dosis TypeFace is coming from "http://cdn.jsdelivr.net/npm/typeface-dosis" 

**Reviewers**
